android/src/main/java/com/reactnativehmssdk/HMSManager.kt

```diff
@@ -16,11 +21,19 @@ import java.util.UUID
 
 @ReactModule(name = REACT_CLASS)
 class HMSManager(reactContext: ReactApplicationContext) :
-    ReactContextBaseJavaModule(reactContext), Application.ActivityLifecycleCallbacks {
+    ReactContextBaseJavaModule(reactContext), Application.ActivityLifecycleCallbacks
+    ,
+    LifecycleEventListener
```

Comment:
remove `LifecycleEventListener`

Reply:
done in commit - f105fe3d91ad50cf3916a645b8e93b524b8c33c1